### DISCUSSION TITLE
Display times with HA selected timezone?

### DISCUSSION THREAD

**Discussion by dmshimself** [dbuezas/lovelace-plotly-graph-card#305] [Q&A] [answered]
Now that HA allows you to select the timezone you wish to use, is it possible to get plotly to use that?  The HA generic history graph does this automatically, but I wonder if there is a way to tweak a plotly card to do the same?

**Comment by dbuezas** [comment]
Good point I'll add this to the next release

**Comment by dbuezas** [comment, marked as answer]
Follow in #306

**Comment by JavanXD** [comment]
<img width="433" alt="image" src="https://github.com/user-attachments/assets/01b4d838-0938-421b-94c9-667beaac87da">

I am still interested in the feature. Even better would be if you could define a default time-zone.

Currently, I am encountering a bug where, if my computer’s timezone is set to a different one (as I recently traveled to the US), but the data is in my original timezone, the times displayed on the Home Assistant card are incorrect. The card uses my computer’s timezone instead of the actual times from the datapoints.
